assets/js/base/components/block-error-boundary/block-error.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { WC_BLOCKS_ASSET_URL } from '@woocommerce/block-settings';
+
+const BlockError = () => {
```

Comment:
I would have the `BlockError` accept a src prop for the image (and default to what you have currently).  Also having explicitly set to an empty string (or null?) would show the message without the image.

I would also have this component accept props for the header and text in the error container for customization defaulting to what you have now.

I would also have a prop for receiving the actual error message to display in it's own container and if that is set and non-empty display it along with the already existing text.

Reply:
Sounds good. Done in c96556276ca0ee8b69f016f9069c47813e341e3f.